.openshift-ci/drivers/scripts/lib.sh

```diff
@@ -44,7 +44,7 @@ get_branch() {
     # ref: https://github.com/kubernetes/test-infra/blob/master/prow/jobs.md#job-environment-variables
     if [[ -n "${CLONEREFS_OPTIONS}" ]]; then
         local base_ref
-        base_ref="$(jq -r '.refs[0].base_ref' <<< "${CLONEREFS_OPTIONS}")" || die "invalid CLONEREFS_OPTIONS json"
+        base_ref="$(jq -r '.refs[] | select(.repo=="collector") | .base_ref' <<< "${CLONEREFS_OPTIONS}")" || die "invalid CLONEREFS_OPTIONS json"
```

Comment:
Can you explain this change?

Reply:
Here is an example of CLONEFEFS_OPTIONS 

`{
  "src_root": "/go",
  "log": "/dev/null",
  "git_user_name": "ci-robot",
  "git_user_email": "ci-robot@openshift.io",
  "refs": [
    {
      "org": "openshift",
      "repo": "release",
      "repo_link": "https://github.com/openshift/release",
      "base_ref": "master",
      "base_sha": "c72dab7b4c721317cbf0808c7a7b0c3d19ef4120",
      "base_link": "https://github.com/openshift/release/commit/c72dab7b4c721317cbf0808c7a7b0c3d19ef4120",
      "pulls": [
        {
          "number": 30786,
          "author": "JoukoVirtanen",
          "sha": "e6d67c54c276a22de1f3bba1c5ae0c5a22f00535",
          "link": "https://github.com/openshift/release/pull/30786",
          "commit_link": "https://github.com/openshift/release/pull/30786/commits/e6d67c54c276a22de1f3bba1c5ae0c5a22f00535",
          "author_link": "https://github.com/JoukoVirtanen"
        }
      ]
    },
    {
      "org": "stackrox",
      "repo": "collector",
      "base_ref": "ROX-11950-collector-ci-migration-update-support-packages",
      "workdir": true
    }
  ],
  "fail": true
}
`

The old way would return master, which is not what we want. The new way returns ROX-11950-collector-ci-migration-update-support-packages, which is correct.